rpc/params/http.go

```diff
@@ -8,6 +8,12 @@ import (
 	"fmt"
 )
 
+const (
+	// JujuCharmURLHeader is the header used to specify the charm URL in a
+	// request.
+	JujuCharmURLHeader = "Juju-Curl"
+)
```

Comment:
I'm shocked this didn't already exist!

This const could do with being used in a few other places. A quick grep has shown:
- api/client/charms/localcharmputters.go
- api/controller/migrationtarget/client.go